documentation/src/main/asciidoc/topics/con_server_hinting.adoc

```diff
@@ -1,25 +1,26 @@
 [id='server-hinting_{context}']
 = Server hinting
 
-The following topology hints can be specified:
-
-Machine::
-  This is probably the most useful, when multiple JVM instances run on the
-  same node, or even when multiple virtual machines run on the same physical machine.
-Rack::
-  In larger clusters, nodes located on the same rack are more likely to experience a
-  hardware or network failure at the same time.
-Site::
-  Some clusters may have nodes in multiple physical locations for extra resilience.
-  Note that Cross site replication is another alternative for
-  clusters that need to span two or more data centres.
-
-All of the above are optional.
-When provided, the distribution algorithm will try to spread the ownership of each segment across as many sites, racks, and machines (in this order) as possible.
+Server hinting increases availability of data in distributed caches by replicating entries across as many servers, racks, and data centers as possible.
+
+NOTE: Server hinting applies only to distributed caches because it distributes data across the physical topology of your {brandname} cluster.
+
+Server hinting requires a site to have multiple racks, a rack has multiple machines, and a machine has multiple {brandname} nodes.
```

Comment:
I think we can refine this section a little. Something like:

"Server hinting is useful when you deploy {brandname} in one or more data centers that have multiple server racks that contain multiple machines.
If all nodes in the cluster run on the same machine then {brandname} creates copies of cache entries in the same physical location.
Server hinting ensures that {brandname} stores replicas in different locations."